jormungandr/src/settings/start/mod.rs

```diff
@@ -216,6 +218,30 @@ fn generate_network(
         p2p.trusted_peers = Some(command_arguments.trusted_peer.clone())
     }
 
+    p2p.trusted_peers.as_mut().map(|peers| {
+        *peers = peers.iter().filter_map(|peer| {
+            let address = match multiaddr_resolve_dns(peer.address.multi_address().clone()) {
+                Ok(Some(addr)) => {
+                    info!(logger, "DNS resolved"; "fqdn" => peer.address.multi_address().to_string(), "resolved" => addr.to_string());
+                    addr
+                }
+                Ok(None) => {
+                    warn!(logger, "failed to resolve DNS"; "fqdn" => peer.address.multi_address().to_string());
+                    return None;
+                },
+                Err(e) => {
+                    warn!(logger, "failed to resolve dns"; "fqdn" => peer.address.multi_address().to_string(), "error" => e.to_string());
```

Comment:
Nitpick: use "DNS" capitalization for consistency.

Reply:
same thing for `FQDN`

Reply:
Ah, the "fqdn" is a structural logging keyword, which I think should be lowercase conventionally (and will be uppercased in backends like systemd)